roles/proxmox_provision_vm/README.md

```diff
@@ -1,9 +1,98 @@
 # proxmox_provision_vm
 
-Should apply `proxmox_server` to KVM to install promoxer
+Should apply` proxmox_server` to KVM to install promoxer
 
-- `add_local_host`: adds host name and ip pair to local /etc/hosts
-- `pve_kvm_sshkeys`: defines SSH keys to be installed on KVM host
-- `kvm_start`: Start server and wait for it to start
+-` proxmox_server`: Dict of proxmox servers and credentials
+    -` key`: Hostname of proxmox server
+        -` api_host`: address of server
+        -` api_user`: username to login as
+        -` api_password`: password to login as
 
-TODO: document other vars and dicts in `tasks/main.yml` and defaults.
+-` add_local_host`: adds host name and ip pair to local /etc/hosts
+-` pve_kvm_sshkeys`: defines SSH keys to be installed on KVM host
+-` node`: hostname listed in proxmox_server
+-` kvm_start`: Start server and wait for it to start
+-` kvm_id`: Assigns ID in proxmox 206
+-` kvm_net_0:` Defines first interface using proxmox format (IE "virtio=00:50:00:5a:55:55,bridge=vmbr0")
+-` kvm_ip_address`: Defines ipv4 address in cloud-init
+-` kvm_gw_address`: Defines ipv4 gateway in cloud-init
+-` kvm_ip_address_cidr`: Defines ipv4 CIDR
+-` kvm_gw6_address`: Defines ipv6 address in cloud-init
+-` kvm_ip6_gateway`: Defines ipv6 gateway in cloud-init
+-` kvm_ip6_address_cidr`: Defines ipv6 CIDR
+-` kvm_net_1:` Defines first interface using proxmox format (IE "virtio,bridge=vmbr1")
+-` kvm1_ip_address`: Defines ipv4 address on second interface in cloud-init
+-` kvm1_ip6_address`: Defines ipv6 address on second interface in cloud-init
+
+Following are mappings to the ansible proxmox plugin. See  [community.general.proxmox](https://docs.ansible.com/ansible/latest/collections/community/general/proxmox_module.html) for more details.
+
+` pve_kvm_update` : ('yes') update
```

Comment:
Please change these entries into a markdown list, with correct spacing. For example:

```suggestion
- `pve_kvm_update`: ('yes') update
```